### PR TITLE
[review: round 2]

### PR DESCRIPTION
**Pull request by jag82**
+ reviewed 510

### CODE REVIEW

File: chapters/510_authentication.md [hunk 1]
```diff
@@ -56,94 +56,94 @@ everything with it that's worth knowing, including possible states, and if neces
...
-to the system. Hence the *PKI* needs to be selfcontained and private in order to function 
-independently so that only invited parties can get involved.
-Referring to the statement mentioned above, *data consumer* have as well to be able to verify the 
+to the system. Hence the *PKI* needs to be self-contained and private in order to function 
+independently so that only invited parties can be involved.
+Referring to the statement mentioned above, *data consumers* also have to be able to verify the 
 identity of the *PDaaS*, in order to prevent man-in-the-middle attacks. Addressing this issue 
 basically means, *data consumers* have to verify the certificate presented by the *PDaaS*. This can
-be done in two ways. One is, a certificate has been installed on the *PDaaS* that is certified and 
+be done in two ways. One is a certificate having been installed on the *PDaaS* that is certified and
```
Comment (lucendio) [inline chapters/510_authentication.md:122]: sounds bit bumpy to me. is that the right version?
